crates/turborepo-auth/src/auth/login.rs

```diff
@@ -313,16 +298,16 @@ mod tests {
         let mut options = LoginOptions::new(&ui, &url, &api_client, &login_server);
 
         let token = login(&options).await.unwrap();
-        assert!(!token.exists);
+        assert!(!matches!(token, Token::Existing(..)));
```

Comment:
nit: [`assert_matches!`](https://doc.rust-lang.org/std/assert_matches/macro.assert_matches.html) is a little cleaner.